src/server/dstu2/metadata/services/metadata.service.js

```diff
@@ -49,13 +47,13 @@ let generateCapabilityStatement = (req, config, logger) => new Promise((resolve,
 
 	// Map all the active resources
 	let active_resources = RESOURCES
-		.map(mapResources(profiles))
+		.map(mapResources(profiles.dstu2))
 		.filter(filterResources);
 
 	// Iterate over the active_resources and execute getCount for each one.
 	// req and logger are by no means necessary, but pass them in so the service can
 	// access the logger and see information in the request if necessary for any validation etc.
-	return Promise.all(active_resources.map(resource => resource.getCount(req, logger)))
+	return Promise.all(active_resources.map(resource => resource.getCount(req, logger, { dstu2: true })))
```

Comment:
Oh I see.  I was thinking you'd just pass the string `dstu2` as a third parameter, but I guess it makes sense to pass a context object so you can add other stuff in the future.  Not sure it needs to be a key though, since you would never have both `dstu2` and `stu3` execute on the same request at the same time.

Reply:
I see.  This way you can do a test in your service for `if (context.dstu2) { ... }` without needing to test for explicit strings.  But if we did `{ version: 'dstu2'}` instead you could use a switch statement rather than a series of ifs...  Choices choices.

Reply:
Yea I flipped on this a few times myself. I am thinking of going back to strings and then exporting more things from this module. Right now fhir-server-core only exports a single function that returns an instance of the Server class. I would like to maybe add some constants (which could contain these strings) and other helpful utils. This way someone could do something like this:

```
const { VERSIONS } = require('@asymmetrik/fhir-server-core');

switch (context.version) {
  case VERSIONS.DSTU2:
  case VERSIONS.STU3:
}
```

and never need to worry about what the strings are.